mkdocs.yml

```diff
@@ -192,6 +192,7 @@ nav:
       - Software contributions:
           - docs/contribute/software/index.md
           - Software development environment: docs/contribute/software/dev-environ.md
+          - docs/contribute/software/add-new-metric-provider.md
```

Comment:
Please provide a title here. it will show up in the left sidebar with this title then.
I suggest: "Add a new metrics provider"

Reply:
Alright I will do it.